src/main/java/UI/WindowManager.java

```diff
@@ -19,6 +19,7 @@ public interface WindowManager {
     // reference constants for types of windows
     String REGISTRATION_REFERENCE_KEY = "userRegistration";
     String LOGIN_REFERENCE_KEY = "login";
+    String AVAILABILITY_REFERENCE_KEY = "availability";
```

Comment:
This is an excellent start; hopefully it paves the way for other UI-side people to understand the role of the WindowManager!